keyboards/custommk/evo70_r2/info.json

```diff
@@ -59,88 +59,265 @@
         "levels": 17,
         "pin": "A6"
     },
+    "layout_aliases": {
+        "LAYOUT": "LAYOUT_all"
+    },
+    "layout_aliases": {
+        "LAYOUT_ansi_blocker": "LAYOUT_ansi_blocker"
+    },
+    "layout_aliases": {
+        "LAYOUT_ansi": "LAYOUT_ansi_blocker"
+    },
+    "layout_aliases": {
+        "LAYOUT_iso": "LAYOUT_iso_blocker"
+    },
```

Comment:
```suggestion
    "layout_aliases": {
        "LAYOUT": "LAYOUT_all",
        "LAYOUT_ansi_blocker": "LAYOUT_ansi_blocker",
        "LAYOUT_ansi": "LAYOUT_ansi_blocker",
        "LAYOUT_iso": "LAYOUT_iso_blocker"
    },
```

Reply:
... :raised_eyebrow: I have no idea how *that* happened.